forge/csrc/passes/commute_utils.cpp

```diff
@@ -898,16 +898,17 @@ bool can_commute_past_op(
     return (is_elementwise(op) and op->op_name() != "interleave") or is_quantization_ops(op);
 }
 
-void update_reshape_attr(graphlib::OpNode *reshape, graphlib::Shape new_shape)
+void update_reshape_attr(graphlib::OpNode *op, graphlib::Shape new_reshape_shape)
```

Comment:
Let's add function doc with more details and main purpose of it.

Reply:
@nvukobratTT , it's a WIP PR
Hence I will request review once it is completed.

Reply:
In that case, let's mark it as draft until it's ready. Also, for future PRs, you can open it in draft mode from the get go :))

<img width="373" alt="image" src="https://github.com/user-attachments/assets/fbec1992-f12a-445d-ba6a-ddd686186c5c">